src/categories/dto/category.updateCategory.dto.ts

```diff
@@ -1,18 +1,26 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { Transform } from 'class-transformer';
-import { IsNumber, IsString } from 'class-validator';
+import { IsNotEmpty, IsOptional, IsPositive, IsString } from 'class-validator';
 import { Types } from 'mongoose';
-
 export class UpdateCategoryDto {
-  @ApiProperty()
+  @ApiProperty({
+    type: String,
+    description: 'Category title',
+  })
   @IsString()
+  @IsNotEmpty()
   title: string;
 
   @ApiProperty({ type: String, isArray: true })
   @Transform(({ value }) => value.map((v) => new Types.ObjectId(v)))
   menus: Types.ObjectId[];
 
-  @ApiProperty()
-  @IsNumber()
+  @ApiProperty({
+    type: Number,
+    description: 'Category rank',
+    default: null,
```

Comment:
Add field required: false to tell that is a optional field

Reply:
Done